lib/backtrace_exception.cpp

```diff
@@ -72,13 +75,17 @@ void backtrace_fill_stacktrace(std::string &msg, void *const *backtrace, int siz
 #endif
 }
 
+}  // namespace P4
+
 #ifdef __GLIBC__
 /* DANGER -- overrides for glibc++ exception throwers to include a stack trace.
  * correct functions depends on library internals, so may not work on some versions
  * and will fail with non-GNU libc++. These function are declared in bits/functexcept.h. */
 
 namespace std {
 
+using ::P4::backtrace_exception;
```

Comment:
I don't think we are allowed to bring new things into `std` namespace this way. FWIW, we cannot define the functions as well, but can't we simply qualify `backtrace_exception` inside functions and not extend `std` this way?

Reply:
Fixed.